samples/dotnet/EventHub/local.settings.json.example

```diff
@@ -0,0 +1,9 @@
+{
+    "IsEncrypted": false,
+    "Values": {
+        "AzureWebJobsStorage": "UseDevelopmentStorage=true",
+        "FUNCTIONS_WORKER_RUNTIME": "dotnet",
+        "BrokerList": "<YOUR_EVENTHUB_NAMESPACE_NAME>.servicebus.windows.net:9093",
```

Comment:
Should we change this to <YOUR_EVENTHUB_NAMESPACE_NAME>?

Reply:
The user might not be able to understand what to fill here.